blockchain/types/transaction_test.go

```diff
@@ -179,6 +209,93 @@ func TestEIP2930Signer(t *testing.T) {
 	}
 }
 
+// This test checks signature operations on access list transactions.
+func TestLondonSigner(t *testing.T) {
+	var (
+		key, _  = crypto.HexToECDSA("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291")
+		keyAddr = crypto.PubkeyToAddress(key.PublicKey)
+		signer1 = NewLondonSigner(big.NewInt(1))
+		signer2 = NewLondonSigner(big.NewInt(2))
+		tx0     = NewTx(&TxInternalDataDynamicFee{AccountNonce: 1, ChainID: new(big.Int)})
+		tx1     = NewTx(&TxInternalDataDynamicFee{ChainID: big.NewInt(1), AccountNonce: 1, V: new(big.Int), R: new(big.Int), S: new(big.Int)})
+		tx2, _  = SignTx(NewTx(&TxInternalDataDynamicFee{ChainID: big.NewInt(2), AccountNonce: 1}), signer2, key)
+	)
+
+	tests := []struct {
+		tx             *Transaction
+		signer         Signer
+		wantSignerHash common.Hash
+		wantSenderErr  error
+		wantSignErr    error
+		wantHash       common.Hash // after signing
+	}{
+		{
+			tx:             tx0,
+			signer:         signer1,
+			wantSignerHash: common.HexToHash("b6afee4d44e0392fb5d3204b350596d6677440bced7ebd998db73c9671527c57"),
+			wantSenderErr:  ErrInvalidChainId,
+			wantHash:       common.HexToHash("a2c6373b7eed946fd4165a0d8503aa26afc8e99f09e2be58b332fbbedc279f7a"),
```

Comment:
```suggestion
```

Reply:
Maybe we need to add wantSignErr or remove this? wantHash seems like unused.

Reply:
This hash data used in test. In L276 SignTx() will set all of field which need to calculate hash

Reply:
Ah, i misunderstood the test logic. Thank you. :)